webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricCriteria.js

```diff
@@ -37,6 +37,9 @@ export class SakaiRubricCriteria extends RubricsElement {
       this.querySelector(`[data-criterion-id="${e.detail.data.criterionId}"] .drag-handle`).focus();
     });
 
+    // Reordering doesn't really care about the weight changes, but the event does get the criteria to update in the parent rubric object
```

Comment:
have you tried this.requestUpdate()

Reply:
I did, and it didn't work.  I'm guessing because the "parent" doesn't know anything about the updated criteria?

Reply:
Let me poke at it again...